internal/tweets/create.go

```diff
@@ -13,45 +12,63 @@ import (
 	"github.com/go-playground/validator/v10"
 )
 
+const maxLengthTweet = 400
```

Comment:
this should not be const by now? it was moved to settings?

Reply:
in TWI-30. it's still opened.